scripts/PSRA_runCreate_tables.py

```diff
@@ -0,0 +1,89 @@
+#Import statements
+import sys
+import os
+import argparse
+import configparser
+import psycopg2
+import logging
+import csv
+
+'''
+
+'''
+
+#Main Function
+def main ():
+    logging.basicConfig(level=logging.INFO,
+                format='%(asctime)s - %(levelname)s - %(message)s',
+                handlers=[logging.FileHandler('/tmp/{}.log'.format(os.path.splitext(sys.argv[0])[0])),
+                            logging.StreamHandler()])
+    os.chdir(sys.path[0])
+    auth = get_config_params('config.ini')
+    args = parse_args()
+    
+    with open("/usr/src/app/cHazard/{prov}/cH_{prov}_hmaps.csv".format(**{'prov':args.province}), "r") as f:
+        reader = csv.reader(f)
+        hmapColumns = next(reader)
+    hmapColumns = ','.join('"{0}"'.format(w) for w in hmapColumns)
+    hmapColumns = hmapColumns.replace('-','_')
+    hmapColumns = hmapColumns.replace('"lon"','lon')
+    hmapColumns = hmapColumns.replace('"lat"','lat')
+    # hmapColumns = hmapColumns.replace('"PGA_0.02"','PGA_0.02')
+    # hmapColumns = hmapColumns.replace('"PGA_0.1"','PGA_0.02')
+
+    with open("/usr/src/app/cHazard/{prov}/cH_{prov}_uhs.csv".format(**{'prov':args.province}), "r") as f:
```

Comment:
Can these paths be relative to the app directory? Something to try perhaps in the next version.

Reply:
Yes, that wouldn't be too difficult